.travis.yml

```diff
@@ -53,6 +53,7 @@ jobs:
     #- env: DB=mysql; MW=REL1_31; SESAME=2.8.7; PHPUNIT=6.5.*
     - env: DB=mysql; MW=REL1_33; ES=5.6.6; PHPUNIT=6.5.*
     - env: DB=mysql; MW=REL1_34; PHPUNIT=6.5.*
+    - env: DB=mysql; MW=REL1_31; BLAZEGRAPH=1.5.2; PHPUNIT=6.5.*
```

Comment:
Strange that this is failing all of a sudden. I am pretty sure not much was touched here lately.